### PR TITLE
Add junk to README

### PR DESCRIPTION
**Pull request by kylebrain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ An app to help you safely navigate winter weather roads.
 5. Run ```npm install```
 6. Run ```npm install -g expo-cli```
 7. Run ```npm start``` or ```expo start```
+
+EYY WOAAHHH
```
Comment (kylebrain) [inline README.md:19]: THIS IS WRONG BAD BOO

**Review by ErylKenner — APPROVED** [review]

**Review by kylebrain — COMMENTED** [review]
This is my comment

**Review by michaelmartinelli15 — CHANGES_REQUESTED** [review]
u r dum

**Review by GarrettMoore22 — APPROVED** [review]

**Comment by kylebrain** [comment]
Closing cause bad

**Pull request closed without merging**